librz/core/cmd/cmd_search_rop.c

```diff
@@ -727,3 +730,389 @@ static void rop_classify(RzCore *core, Sdb *db, RzList /*<char *>*/ *ropList, co
 
 	free(str);
 }
+
+RZ_API void rz_rop_constraint_free(void *data) {
```

Comment:
For all `*_free()` functions we usually allow the argument to be NULL and add a notation `RZ_NULLABLE` to that argument.